core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -2690,7 +2690,10 @@ object SparkContext extends Logging {
         setActiveContext(new SparkContext(config))
       } else {
         if (config.getAll.nonEmpty) {
-          logWarning("Using an existing SparkContext; some configuration may not take effect.")
+          logWarning("Using an existing SparkContext; some configuration may not take effect." +
+            " For how to set these configuration correctly, you can refer to" +
+            " https://spark.apache.org/docs/latest/configuration.html" +
```

Comment:
Is it okay to always refer to the `latest` page?

Reply:
> Is it okay to always refer to the `latest` page?

Hmmmm, for current spark version maybe better?

Reply:
yea, I think so if possible.

Reply:
Yeah, if possible that would be great. See also `SparkVersion` expression.

Reply:
Yea, updated.
Current demo warn log is 
```
13:23:48.133 WARN org.apache.spark.sql.SparkSession$Builder: Since spark has been submitted, such configurations `spark.driver.memory -> 1g` may not take effect. For how to set these configuration correctly, you can refer to https://spark.apache.org/docs/3.2.0-SNAPSHOT/configuration.html#dynamically-loading-spark-properties.

```

Reply:
It seems you need to drop the suffix: `3.2.0-SNAPSHOT` -> `3.2.0`

Reply:
> It seems you need to drop the suffix: `3.2.0-SNAPSHOT` -> `3.2.0`

Done